src/jaxsim/api/contact.py

```diff
@@ -303,6 +301,58 @@ def in_contact(
     return links_in_contact
 
 
+def estimate_good_contacts_parameters(
+    model: js.model.JaxSimModel,
+    *,
+    standard_gravity: jtp.FloatLike = jaxsim.math.StandardGravity,
+    static_friction_coefficient: jtp.FloatLike = 0.5,
+    number_of_active_collidable_points_steady_state: jtp.IntLike = 1,
+    damping_ratio: jtp.FloatLike = 1.0,
+    max_penetration: jtp.FloatLike | None = None,
+    **kwargs,
+) -> jaxsim.rbda.contacts.ContactParamsTypes:
+    """
+    Estimate good contact parameters.
+
+    Args:
+        model: The model to consider.
+        standard_gravity: The standard gravity constant.
+        static_friction_coefficient: The static friction coefficient.
+        number_of_active_collidable_points_steady_state:
+            The number of active collidable points in steady state supporting
+            the weight of the robot.
+        damping_ratio: The damping ratio.
+        max_penetration:
+            The maximum penetration allowed in steady state when the robot is
+            supported by the configured number of active collidable points.
+        kwargs:
+            Additional model-specific parameters passed to the builder method of
+            the parameters class.
+
+    Returns:
+        The estimated good soft contacts parameters.
+
+    Note:
+        This is primarily a convenience function for soft-like contact models.
+        However, it provides with some good default parameters also for the other ones.
+
+    Note:
+        This method provides a good set of contacts parameters.
+        The user is encouraged to fine-tune the parameters based on the
+        specific application.
+    """
+
+    return estimate_good_soft_contacts_parameters(
+        model=model,
+        standard_gravity=standard_gravity,
+        static_friction_coefficient=static_friction_coefficient,
+        number_of_active_collidable_points_steady_state=number_of_active_collidable_points_steady_state,
+        damping_ratio=damping_ratio,
+        max_penetration=max_penetration,
+        **kwargs,
+    )
```

Comment:
Does this method differs in any way from `estimate_good_soft_contacts_parameters`? If I understood well, this should replace the aforementioned method in future releases, so probably we could start a deprecation cycle in which we can rename `estimate_good_soft_contacts_parameters` to `_estimate_good_soft_contacts_parameters`, and then implement `estimate_good_soft_contacts_parameters` as:

```python
def estimate_good_soft_contacts_parameters(*args, **kwargs):
    logging.warning("This method is deprecated, please use `estimate_good_contact_parameters`)
    return estimate_good_contacts_parameters(*args, **kwargs)
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/260/commits/8c884258469359bd4303151aa4faeba5262d847e.